task.sql

```diff
@@ -1,2 +1,14 @@
-# Write your SQL code for the database creation here. Good luck! 
-USE ShopDB; 
+USE ShopDB;
+
+-- Create an index for the Customers table to optimize login by Email
+CREATE INDEX Email ON Customers (Email);
```

Comment:
Consider using a more descriptive name for the index on the `Email` column, such as `idx_Customers_Email`, to make it clear which table and column the index is associated with. This can improve maintainability and readability of the database schema.